scripts/display-travis-status.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const debug = require('debug')('display_travis_status')
+const pollTravis = require('../lib/pollTravis')
+
+module.exports = function (app) {
+  app.on('pull_request.opened', (event) => {
+    const owner = event.repository.owner.login
+    const repo = event.repository.name
+    if (repo !== 'nodejs.org') return
```

Comment:
I agree with the sentiment of having one bot to rule them all. It will raise requirements for the software (needs to store state and likely other stuff) as well as security surrounding it since it will contain access keys to github and jenkins. If the bot turns up being adopted by nodejs org (has it already?) the build team is happy to arrange deployment.